metadata/db.go

```diff
@@ -76,15 +81,15 @@ type DB struct {
 
 // NewDB creates a new metadata database using the provided
 // bolt database, content store, and snapshotters.
-func NewDB(db *bolt.DB, cs content.Store, ss map[string]snapshots.Snapshotter) *DB {
+func NewDB(db *bolt.DB, policy string, cs content.Store, ss map[string]snapshots.Snapshotter) *DB {
```

Comment:
We try to avoid making long signatures or adding configuration options. We should probably consider doing the variadic options approach here.

Reply:
Nice point. And I think it's necessary since NewDB() is a exported function, while I didn't do the same thing to the newContentStore()

pls have a look